types/cheerio/index.d.ts

```diff
@@ -266,7 +268,7 @@ interface CheerioElement {
 }
 
 interface CheerioAPI extends CheerioSelector, CheerioStatic {
-  load(html: string, options?: CheerioOptionsInterface): CheerioStatic;
+  load(html: string | Buffer, options?: CheerioOptionsInterface): CheerioStatic;
```

Comment:
I wrote like this by using union type `string | Buffer`. However, the original author may prefer to define separately like the following.

```ts
load(html: string, options?: CheerioOptionsInterface): CheerioStatic;
load(html: Buffer, options?: CheerioOptionsInterface): CheerioStatic;
```

I'd like to change the definition when you suggest.